src/org/labkey/test/pages/core/admin/ProjectSettingsPage.java

```diff
@@ -41,14 +39,27 @@ public static ProjectSettingsPage beginAt(WebDriverWrapper driver, String contai
         return new ProjectSettingsPage(driver.getDriver());
     }
 
-    public void save()
+    public boolean getShouldInherit()
     {
-        elementCache().saveButton.click();
+        return elementCache().shouldInherit.isChecked();
     }
 
-    public Checkbox getDiscussionEnabledCheckbox()
+    public void setShouldInherit(boolean value)
     {
-        return elementCache().discussionEnabled;
+        if(value)
+        {
+            elementCache().shouldInherit.check();
+        }
+        else
+        {
+            elementCache().shouldInherit.uncheck();
+        }
```

Comment:
I think this component has a `set` method.
```suggestion
        elementCache().shouldInherit.set(value);
```

Reply:
Changed.

Reply:
Changed.